src/Menu.cpp

```diff
@@ -472,9 +472,25 @@ void Menu::DrawSettings()
 						ImGui::TextColored(themeSettings.StatusPalette.Error, feat->failedLoadedMessage.c_str());
 					}
 
-					if (!isDisabled && isLoaded) {
+					if (!isDisabled) {
 						if (ImGui::BeginChild("##FeatureConfigFrame", { 0, 0 }, true)) {
-							feat->DrawSettings();
+							if (isLoaded) {
+								// draw settings for loaded feature
+								feat->DrawSettings();
+							} else {
+								// draw any unloaded UI elements like help text about the feature
+								feat->DrawUnloadedUI();
```

Comment:
Does this override the standard no ini available or function is disabled messaging?

Reply:
It does not override, both messages are printed currently

Reply:
Add a default parameter whether to override I guess. So if it's non blank and parameter set, the override can kick in.

Reply:
updated